app/styles/custom-components/_vlc-login.scss

```diff
@@ -0,0 +1,12 @@
+.vlc-login {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
```

Comment:
This only makes it so that it is a tiny bit lower (header height to be exact)|, but I would just get rid of this. Remember, you can see these styles appear in Figma quite frequently, and you should disregard them in most of the cases.

Reply:
I saw, it indeed only changed a little bit.. I need to check in which css class it is centering the component now though as this is unclear to me, as this snippet would center the component.